### PR TITLE
JavaScript Restaurant Page

### PR DESCRIPTION
**Pull request by jamezjaz**
**This is an implementation of DOM manipulation in JavaScript...**
On this branch, I implemented the following:

[x] Aside from the home page, I also created three other pages; menu, about, and index pages.
[x] Each page is created using its own module.
[x] I implemented a function that changes the background image when the page is scrolled.
[x] I also implemented a function that returns the current date on the index page

### CODE REVIEW

**Comment by jamezjaz** [comment]
Dear Reviewer,
I couldn't fix  `no-unused-vars` ESlint error on this app. The app doesn't work when I delete the following variables:

**index.js**
  1:8  error  'navBar' is defined but never used          no-unused-vars
  2:8  error  'footer' is defined but never used          no-unused-vars
  3:8  error  'menu' is defined but never used            no-unused-vars
  4:8  error  'aboutUs' is defined but never used         no-unused-vars
  5:8  error  'contactMe' is defined but never used       no-unused-vars
  6:8  error  'bgChanger' is defined but never used       no-unused-vars
  8:7  error  'index' is assigned a value but never used  no-unused-vars

**navbar.js**
1:8  error  'date' is defined but never used  no-unused-vars.

Help me out if you have an idea on how to solve this problem, or maybe how to disable those errors.
Thank you very much.

File: dist/style.css [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+
+nav {
+  position: sticky;
+  top: 0;
+}
+
+#home a,
+#menu a,
+#about a,
+#contact a {
```
Comment (Dandush03) [inline dist/style.css]: - [ ] __Don't target the container through an ID selector, make sure to use classes__

File: src/bgChanger.js [hunk 2]
```diff
@@ -0,0 +1,10 @@
+function bgChanger() {
+  if (this.scrollY > this.innerHeight / 10) {
+    document.body.classList.add('mainmenu');
+  } else {
+    document.body.classList.remove('mainmenu');
+  }
+}
+window.addEventListener('scroll', bgChanger);
+
+export default bgChanger;
```
Comment (Dandush03) [inline src/bgChanger.js]: - [ ] __Use It uses ES6 syntax; Arrow Functions, for example:__
>
```javascript
const myModule = (parentNode) => {
    const node = document.createElement('div');
    // Add whichever sub-nodes, ids and style is required

    parentNode.appendChild(node);
};

export { myModule as default };
```

File: package.json [hunk 3]
```diff
@@ -0,0 +1,34 @@
+{
+  "name": "restaurant_page",
+  "version": "1.0.0",
+  "description": "This is a restaurant landing page",
+  "main": "index.js",
+  "scripts": {
+    "test": "\"\""
+  },
```
Comment (Dandush03) [inline package.json]: - [ ] __Necessary run common commands like `build`, `development` should be in `package.json` inside `scripts`. Here is a basic example__
```json
"scripts": {
    "dev": "webpack --mode development",
    "build": "webpack --mode production",
    "start": "webpack-dev-server --mode development --open" // optional
  },

```

**Review by Dandush03 — CHANGES_REQUESTED** [review]
# Status: :red_circle: Require Changes :red_circle: 
### Great work for first attempt, checkout my comments below
### Next submit be sure your EsLint throw no error 
# Happy Coding

**Comment by Dandush03** [comment]
# Status: Approved :rocket: 
### Great Work, Keep it up!
# Happy Coding :tophat: :muscle:

**Comment by jamezjaz** [comment]
Thank you Dan.
You're appreciated

**Pull request merged**